business_requirement/views/business_view.xml

```diff
@@ -0,0 +1,149 @@
+                        <button name="action_button_cancel" string="Cancel" type="object" attrs="{'invisible': [('state', 'in', ('done','cancel'))]}" groups="business_requirement.group_business_requirement_user" class="oe_highlight"/>
+                        <field name="state" widget="statusbar" statusbar_visible="draft,confirmed,approved,done,cancel"/>
+                    </header>
+                    <sheet>
+                        <h1>
+                            <field name="name" class="oe_inline" readonly="1"/>
+                        </h1>
+                        <group>
+                            <group>
+                                 <field name="description"/>
+                             </group>
+                        </group>
+                        
+                        <notebook>
+                            <page string="Business Requirement">
+                                <group string="Customer Story">
+                                     <field name="business_requirement" widget="html" nolabel="1"/>
+                                </group>
+                                <group string="Scenario">
+                                    <field name="scenario" widget="html" nolabel="1"/>
+                                </group>
+                                <group string="Gap">
+                                     <field name="gap" widget="html" nolabel="1"/>
+                                </group>
+                            </page>
+                            <page string="Sub Business Requirement">
+                                <field name="business_requirement_ids" readonly="1" context="{'form_view_ref':'view_business_requirement_form', 'tree_view_ref':'view_business_requirement_tree'}"/>
+                            </page>
+                            <page string="Other Information">
+                                <group col="4">
+                                    <field name="categ_id"/>
+                                    <field name="parent_id"/>
+                                    <field name="level"/>
+                                    <field name="drop"/>
+                                    <field name="change_request"/>
+                                </group>
+                            </page>
+                        </notebook>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+
+        <record id="view_business_requirement_tree" model="ir.ui.view">
+            <field name="name">business.requirement.tree</field>
+            <field name="model">business.requirement</field>
+            <field name="arch" type="xml">
+                <tree string="Business Requirement">
+                    <field name="sequence"/>
+                     <field name="name"/>
+                     <field name="description"/>
+                     <field name="categ_id"/>
```

Comment:
@dreispt Would not be better to add change request in a separate module (link to the @jbeficent module?) or later on. All the more that those modules have not  yet landed in OCA AFAIK